src/SearchResults.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import HeadingRow from "./HeadingRow";
+import BodyRow from "./BodyRow";
+import CustomerProfile from "./CustomerProfile";
+import moment from "moment";
+
+const SearchResults = ({ results, setBooking }) => {
+  const [customerId, setCustomerId] = useState(null);
+  const [isSorted, setIsSorted] = useState({
+    title: false,
+    firstName: false,
+    surname: false,
+    email: false,
+    roomId: false,
+    checkInDate: false,
+    checkOutDate: false,
+    numOfNight: false
+  });
+
+  const sortBookings = event => {
+    let sortedBookings = [];
+    let reversedBookings = [];
+    let key = event.target.title;
+
+    console.log(isSorted[key]);
+    if (!isSorted[key]) {
+      if (key == "roomId") {
+        sortedBookings = results.sort((a, b) => a[key] - b[key]);
+      } else if (key == "numOfNight") {
+        sortedBookings = results.sort(
+          (a, b) =>
+            moment(a.checkOutDate).diff(a.checkInDate, "days") -
```

Comment:
You're calculating the number of nights twice in your app - here and in the `BodyRow` component again so there is some unnecessary code repetition. You could add a `numOfNight` property to each element of the bookings array earlier and then just render it or use for sorting without having to calculate it. Try to think where would be the best place to do that🙂